src/content/mod.rs

```diff
@@ -24,19 +24,24 @@ pub enum Error {
     UnexpectedDataType,
     #[cfg(feature = "_cargo_insta_internal")]
     MissingField,
+    #[cfg(feature = "_cargo_insta_internal")]
+    FileIo(std::io::Error, std::path::PathBuf),
 }
 
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        use Error::*;
```

Comment:
I don't like this pattern. It's very error prone. Please leave it with the explicit namespacing.